app/uk/gov/hmrc/customs/emailfrontend/views/helpers/inputDate.scala.html

```diff
@@ -0,0 +1,86 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(formItem:Form[_], fieldName: String, args: (Symbol,Any)*)(implicit lang: Messages)
+
+@import uk.gov.hmrc.play.views.html.{ helpers => playUi}
```

Comment:
Can you inject 'playUi' using guice instead of static import, as it is deprecated as of play 2.6